x-pack/plugins/observability_solution/observability/public/pages/alert_details/components/header_actions.tsx

```diff
@@ -52,12 +61,18 @@ export function HeaderActions({
     },
     triggersActionsUi: { getEditRuleFlyout: EditRuleFlyout, getRuleSnoozeModal: RuleSnoozeModal },
     http,
+    application: { navigateToApp },
+    investigate,
   } = useKibana().services;
 
   const { rule, refetch } = useFetchRule({
     ruleId: alert?.fields[ALERT_RULE_UUID] || '',
   });
 
+  const { data: investigations } = useFetchInvestigationsByAlert({
```

Comment:
I think we should not even try to fetch the investigations in case the investigate feature flag is not enabled

Reply:
Good catch. Updated in 5a24c0b2cccf8e1e7cc97ea1d739e683002ef032